apps/allocations/package.json

```diff
@@ -51,6 +51,7 @@
     "@babel/plugin-proposal-class-properties": "^7.4.4",
     "@babel/preset-env": "^7.4.5",
     "@babel/preset-react": "^7.0.0",
+    "@tps/test-helpers": "^0.0.1",
```

Comment:
`@tps/test-helpers` is not a real repository. For now, we're requiring files explicitly from the `PROJECT_ROOT/shared` directory, using terrible file paths like `import X from '../../../../../../shared/utils'`

Reply:
Right, I was getting this error without it `Error: Could not find @tps/test-helpers/contracts/factory/EVMScriptRegistryFactory.sol from any sources; imported from .../open-enterprise/apps/allocations/test/Spoof.sol`. The test references `@tps/test-helpers/artifacts.js` to import a few contracts. I added `@tps/test-helpers` to package.json and then called `npm link ../shared/test-helpers`. Don't know if that is how we want to do things but it solved the problem. Also, it seems the only real blocker is the StandardBounties contract, everything else can be imported from `@aragon` packages. We could include a StandardBounties package, it looks like there is one available on npm: https://www.npmjs.com/package/bounties

Reply:
@chadoh I just attempted to replace references to `@tps` with relative paths e.g. `../../shared` which works fine until I try to run a coverage test. Because solidity-coverage copies the files over to `coverageEnv` the relative paths are no longer correct

Reply:
Hmm, I'm not sure how to do this with contracts. @topocount @ottodevs can you advise?

I don't think `npm link` will work, as that is local to your machine.

Reply:
I commented on this, please let me know if something is not clear